code/modules/projectiles/guns/misc/medbeam.dm

```diff
@@ -41,6 +43,15 @@
 		on_beam_release(current_target)
 	current_target = null
 
+//and this is the proc that is called when the beam fails due to something, this proc triggers when the beam qdels and calls losetarget so don't losetarget BEFORE this
```

Comment:
this isn't the right format for a dmdoc comment, needs to be triple slashes.

Reply:
See above